### PR TITLE
Add Moose 7.0

### PR DESCRIPTION
**Pull request by badetitou**
Hi,

I've tried to change the files to add Moose 7.0
It's the first time i'm doing something like this, so it needs a real review.

### CODE REVIEW

**Comment by fniephaus** [comment]
Hi @badetitou,
Thanks for the PR. Looks good so far. However, we are deprecating image names without architecture suffixes (for 32/64 bit), so we don't need to support Moose-7.0, only Moose32-7.0 and Moose64-7.0. BTW: are there 64bit Moose images?

**Comment by badetitou** [comment]
There is a Moose 64 bit image. But I don't know if it's for public usage currently.
I would prefer to create a second pull request for Moose 64 later.

So, I suppose I should remove the image without architecture ?

File: pharo/run.sh [hunk 1]
```diff
@@ -79,7 +79,10 @@ moose::get_image_url() {
 
   case "${smalltalk_name}" in
     "Moose32-trunk"|"Moose-trunk")
-      moose_name="moose-6.1"
+      moose_name="moose-7.0"
+      ;;
+    "Moose32-7"*|"Moose-7*")
```
Comment (fniephaus) [inline pharo/run.sh]: s/"Moose-7*"/"Moose-7"* and this can be merged with the below case for Moose 6.

**Comment by fniephaus** [comment]
Let's leave the Moose image without architecture for now. Sure, 64bit images can be added later.

**Comment by badetitou** [comment]
I've fixed the code as you proposed @fniephaus

**Pull request merged**